net/openfortivpn/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        adrienverge openfortivpn 1.6.0 v
```

Comment:
1.7.0 is out, so why not use that?

Reply:
The build for 1.7.0 was broken on Mac OS. That has been fixed now (https://github.com/adrienverge/openfortivpn/issues/337). I'll update to 1.7.1